Gemfile

```diff
@@ -6,6 +6,7 @@ eval_gemfile(File.expand_path("gems/pending/Gemfile", __dir__))
 
 gem "activerecord-deprecated_finders", "~>1.0.4",  :require => "active_record/deprecated_finders"
 gem "rails",                           "~>4.2.5"
+gem "activerecord-session_store"
```

Comment:
Can you lock to a specific version with "~>" so we don't pull in future ones by accident?